<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1539</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_1">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GRAY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HLS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>YUV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LUV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge Detection</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>100</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Regular Thresholding</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>210</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Contour Detection</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel X</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="videofeed">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>                                                                                                Video Feed 1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Color Space</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Webcam/Source</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>0</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>  Filters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="videofeed2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>320</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>                                                                                        Video Feed 2</string>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QDial" name="dial_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QDial" name="dial_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QDial" name="dial_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>EDGES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TH_MIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>COLOR-TH1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TH_MAX</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Thresholding Mode  </string>
    </property>
    <property name="indent">
     <number>-2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dial_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>746</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COLOR-TH3</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_6">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dial_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>COLOR-TH2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>160</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Thresholding</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
    <property name="indent">
     <number>-2</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1539</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dial</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_3</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_4</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
